packages/bridge/src/vl/legends/index.ts

```diff
@@ -18,7 +18,7 @@ function _getColorValue(viz, propName) {
     return rgbaToString(_getLegendData(prop));
   }
 
-  if (propName === 'color' || propName === 'strokeColor') {
+  if (prop.type === 'color') {
```

Comment:
I think I'd need more context on this change :)

Reply:
Now that we're allowing to use a VL variable, we should check if the property from the viz (the style property, or the variable) has a 'color' type. For example, having:

```
@my_var: blue
color: blue
` ``

Both `@my_var` and `color` type will be color.